src/main/kotlin/app/Configurator.kt

```diff
@@ -4,163 +4,116 @@
 package app
 
 import app.utils.Options
-import app.utils.UsernameValidator
-import org.apache.commons.codec.digest.DigestUtils
-import java.io.File
+import app.utils.PasswordHelper
+import com.fasterxml.jackson.core.JsonParseException
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.KotlinModule
 import java.io.IOException
```

Comment:
We are using Google Conventions for Java/Kotlin code. It says that one should ASCII sort imports.